Shruti Bist/Assignment-2/basic-auth/authentication/forms.py

```diff
@@ -0,0 +1,8 @@
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth.models import User
+
+
+class CreateUserForm(UserCreationForm):
+    class Meta:
+        model = User
+        fields = ['email', 'password']
```

Comment:
email is not a defined field in UserCreationForm , you'll have to create it statically and then add it in fields.
Refer this for help:
https://stackoverflow.com/questions/48049498/django-usercreationform-custom-fields